src/components/data-exploration/generic-gene-table/ComponentActions.jsx

```diff
@@ -60,7 +60,7 @@ const ComponentActions = (props) => {
   }
 
   return (
-    <Dropdown arrow type='link' size='small' menu={menu} trigger={['click']}>
+    <Dropdown arrow type='link' size='small' overlay={menu} trigger={['click']}>
```

Comment:
Doesnt it say that overlay is being depricated? You should be using menu

Reply:
if I use menu I need to perform more refactoring tha twill potentially break something, I am keeping this with minimal changes so I don't need to worry about changing functiionality since we want this merged i fast